src/main/java/com/gmail/goosius/siegewar/enums/SiegeStatus.java

```diff
@@ -72,6 +72,35 @@ public String getName() {
 		return this.name;
 	}
 
+	public boolean allowsPlundering() {
+		return attackersWon();
+	}
+
+	public boolean allowsInvading() {
+		return attackersWon();
+	}
+
```

Comment:
Following on from the separate suggests to add a whoGetsWarchest() method, and a getPlunderReductionPercentage() method --->

If both of those are added, then it would presumably be logical to also add a getWarchestReductionPercentage() method

Reply:
WarChest has been tweaked here https://github.com/TownyAdvanced/SiegeWar/pull/792/commits/82857329210d18de91a53273eb7256e42d701332